### ISSUE TITLE
"Open With WLinux" missing in Context Menu.

### ISSUE THREAD

**Issue by muygalan** [WhitewaterFoundry/WLinux#242]
**Describe the bug**
I have tried enabling the **EXPLORER** add-on to enable right-click on folders in Windows Explorer to open them in Wlinux, but the option never appears when I right click the white space in folders. 

**To Reproduce**
1. Open Wlinux execute `wlinux-setup`
2. Press the return key for 3 consecutive prompts.
3. Select the **EXPLORER** option
4. Press the return key for 2 consecutive prompts.
5. Open an arbitrary folder and right-clicking white space. (I also have tried shift right-clicking)
6. "Open With Linux" option (or equivalent) missing entirely.

**Expected behavior**
When I right-click or shift + right-click, I expect the folder to open as the current directory in WLinux.

**Screenshots**
![image](https://user-images.githubusercontent.com/10903350/49332550-eade7f00-f57c-11e8-9d8c-2608287113e9.png)

**Additional context**
Although you may see an option which reads "Open Linux Bash", please note this was added manually recently after following a tutorial online. It does not launch **wlinux**; please disregard. 

**Basic Troubleshooting Checklist**

[x] I have searched Google for the error message.

[x] I have checked official WSL troubleshooting documentation: https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#confirm-wsl-is-enabled.

[x] I have searched the official Microsoft WSL issues page: https://github.com/Microsoft/WSL/issues.

[x] I have searched the WLinux issues page: https://github.com/WhitewaterFoundry/WLinux/issues.

I have seen similar issues concerning this feature, but they are not specific to mine.

[x] I have reset WLinux: Settings->Apps->Apps & features->WLinux->Advanced Options->Reset. 

[x] I have disabled and re-enabled WSL in Windows Features.

[x] I have run Windows 10 updates and restarted.

What other troubleshooting have you attempted?

Insert here:

**WLinux Version**

Find: Settings->Apps->Apps & features->WLinux->Advanced Options->Version.

Insert here: 1.1.25.0

**Windows Build**

Run 'systeminfo | findstr /C:"OS"' in Command Prompt and insert here:

OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.17134 N/A Build 17134
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
BIOS Version:              American Megatrends Inc. P05ADH.017.140421.SH, 4/21/2014

For help on retrieving: https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#check-your-build-number

**Comment by sirredbeard** [comment]
Do you happen to have a space in your Windows username?

**Comment by muygalan** [comment]
> Do you happen to have a space in your Windows username?

@sirredbeard I do not.

**Comment by sirredbeard** [comment]
No, that's the way to update wlinux-setup. We will look into this.

**Comment by muygalan** [comment]
@crramirez Do you mean launch wlinux and type: `wlinux-setup set -x` ?

If yes, I pressed "OK" for the first 2 prompts. 

- When asked if I would like to download and restart, I selected NO. 
- A prompt appeared stating it was continuing with built-in Wlinux-setup, I selected OK.
- I selected the EXPLORER and it brought me back to the terminal with the following message:

```
Starting wlinux-setup.
Applying Hotpatches
1: Applying temporary workaround for bug #199
alias clear="clear -x"
Selected:
```

If I am not fulfilling your request properly, kindly clarify a bit further. Thank you.

**Comment by crramirez** [comment]
Hello,

What I mean is: edit the file /etc/setup and add set -x after the first
line. Save the file and then execute wlinux-setup and say No when it ask
for an update.

Regards

**Comment by muygalan** [comment]
@crramirez Pardon the delay. Could you advise on how I edit the file? I have wlinux pinned to my Windows Star Menu, but when I right-click the tile/file it does not provide an option such as 'properties'.

![image](https://user-images.githubusercontent.com/10903350/49705154-4dbdbf00-fbea-11e8-979e-580eb3a95a90.png)

**Comment by crramirez** [comment]
Hello,

Edit the file in the same way you edit any file in Linux. Inside WLinux
type sudo nano /etc/setup

Regards

**Comment by muygalan** [comment]
@crramirez Thanks for the additional instructions. Please see below for result:

```
+ RESPONSE=
+ CURDUR=
+ TMPDIR=
+ VERSION=1.1.25
+ GOVERSION=1.11.2
+ DOCKERVERSION=18.09.0
+ DOCKERCOMPOSEVERSION=1.23.1
++ cmd.exe /c 'echo %HOMEDRIVE%%HOMEPATH%'
++ tr -d '\r'
+ wHomeWinPath='C:\Users\muyga'
++ wslpath -u 'C:\Users\muyga'
+ wHome=/mnt/c/Users/muyga
+ '[' '!' -f /home/muygalan/.updated ']'
+ welcomeprompt
+ whiptail --title 'Welcome to WLinux' --msgbox 'Thank you for supporting sustainable independent open source development.\n
WLinux comes with a core set of useful packages pre-installed, such as curl, git, and wslu. \n
wlinux-setup allows you to install additional hand-curated selections for WLinux and configure optional WSL-specific settings. \n
Many more packages are available via the apt package manager and optional pip, npm, and RubyGems package managers.' 15 96
+ continueprompt
+ whiptail --title 'Continue setup?' --yesno 'Would you like to continue wlinux-setup?' 8 45
+ echo 'Starting wlinux-setup.'
Starting wlinux-setup.
+ updatescriptprompt
+ whiptail --title 'Upgrade setup?' --yesno 'Would you like to download and restart with the most recent version of wlinux-setup?\n\nThis is recommended periodically to get new features and fixes.' 9 90
+ whiptail --title 'Release setup' --msgbox 'Continuing with built-in wlinux-setup.\n\nIf you encounter problems, please try updating wlinux-setup.' 9 90
+ securitypatches
+ echo 'Applying Hotpatches'
Applying Hotpatches
+ echo '1: Applying temporary workaround for bug #199'
1: Applying temporary workaround for bug #199
+ alias 'clear=clear -x'
+ echo 'alias clear="clear -x"'
+ sudo tee -a /etc/profile
alias clear="clear -x"
+ installmenu
++ whiptail --title wlinux-setup --checklist --separate-output '\nHand-curated add-ons [SPACE to select, ENTER to confirm]:' 22 99 14 LANGUAGE 'Change default language and keyboard setting in WLinux' off EXPLORER 'Enable right-click on folders in Windows Explorer to open them in WLinux  ' off SHELLS 'Install and configure zsh, csh, and fish' off EDITORS 'Install text editors neovim, emacs, or Visual Studio Code (requires X)' off PYTHONPI 'Install Python 3.7 and download and install latest PyPi' off NODEJS 'Install Node.js and npm' off GO 'Install the latest Go from Google' off RUBY 'Install Ruby using rbenv and optionally install Rails' off DOTNET 'Install .NET Core SDK from Microsoft and optionally install NuGet' off JAVA 'Install the Java OpenJDK and JRE' off POWERSHELL 'Install PowerShell for Linux and/or Azure CLI tools' off GUI 'Install the basics needed for most GUI apps and configure GUI options' off DOCKER 'Install a secure bridge to Docker for Windows' off CASSANDRA 'Install the NoSQL server Cassandra from Apache' off
+ CHOICE=
+ echo Selected:
Selected:
+ [[ '' == *\L\A\N\G\U\A\G\E* ]]
+ [[ '' == *\E\X\P\L\O\R\E\R* ]]
+ [[ '' == *\S\H\E\L\L\S* ]]
+ [[ '' == *\E\D\I\T\O\R\S* ]]
+ [[ '' == *\P\Y\T\H\O\N* ]]
+ [[ '' == *\N\O\D\E* ]]
+ [[ '' == *\G\O* ]]
+ [[ '' == *\R\U\B\Y* ]]
+ [[ '' == *\D\O\T\N\E\T* ]]
+ [[ '' == *\J\A\V\A* ]]
+ [[ '' == *\P\O\W\E\R\S\H\E\L\L* ]]
+ [[ '' == *\G\U\I* ]]
+ [[ '' == *\D\O\C\K\E\R* ]]
+ [[ '' == *\C\A\S\S\A\N\D\R\A* ]]
+ byemessage
+ whiptail --title 'Setup is complete.' --msgbox 'You may run wlinux-setup again any time by typing $ wlinux-setup\n\nYou may open a browser link to get help any time by typing: $ wlinux-help' 8 80
+ exit 0
```

**Comment by crramirez** [comment]
Ok run it again but select EXPLORER option so I can see why the option
doesn't get installed

**Comment by crramirez** [comment]
As I can see in your logs and your screenshots you are not selecting any
option. Please press the space bar in the item you want to install to be
sure it is selected and then press enter.

**Comment by muygalan** [comment]
@crramirez That worked. I wasn't hitting space bar when selecting the option. Sorry for the hassle.

**Comment by crramirez** [comment]
👌
